source/guides/network/openssh-server.rst

```diff
@@ -104,32 +104,20 @@ Enable SFTP
 ***********
 
 |CL| *disables* the :abbr:`SFTP (SSH File Transfer Protocol)` subsystem by
-default due to security considerations. To enable the SFTP subsystem, you must
-configure the :abbr:`SSHD (SSH Daemon)` service file.
+default due to security considerations. To enable the SFTP subsystem, you can
+configure the :file:`/etc/ssh/sshd_config` file.
 
-#. Create a systemd drop-in directory for the SSHD service:
-
-   .. code-block:: bash
-
-      sudo mkdir -p /etc/systemd/system/sshd@.service.d
-
-#. Create the following file:
-   :file:`/etc/systemd/system/sshd@.service.d/sftp.conf`
+#. Create the following file, if it does not already exist:
```

Comment:
I think this step 1 should match the step 1 of the root login and x11-forwarding sections. You are doing the exact same thing so I think the wording should match exactly.  I really doubt anyone has to `mkdir /etc/ssh` as described in the root login section and that command would fail anyways without sudo. I recommend changing the root login and x11-forwarding section step 1/2 to match your step 1 in the sftp section.

Reply:
Agree -- fixed in next commit.